content-src/asrouter/asrouter-content.jsx

```diff
@@ -299,14 +312,22 @@ export class ASRouterUISurface extends React.PureComponent {
   render() {
     const {message, bundle} = this.state;
     if (!message.id && !bundle.template) { return null; }
-    return (
-      <React.Fragment>
-        {this.renderPreviewBanner()}
-        {this.renderFirstRunOverlay()}
-        {this.renderOnboarding()}
-        {this.renderSnippets()}
-      </React.Fragment>
-    );
+
+    if (!TEMPLATES_BELOW_SEARCH.includes(message.template)) {
```

Comment:
Personally i'd use an affirmative variable for this like 

```js
render() {
  const {message, bundle} = this.state;
  if (!message.id && !bundle.template) { return null; }
  const shouldRenderBelowSearch = TEMPLATES_BELOW_SEARCH.includes(message.template);

  return shouldRenderBelowSearch ?
    // Render special below search snippets in place;
    this.renderSnippets() :
    // For onboarding, regular snippets etc. we should render
    // everything in our footer container.
    ReactDOM.createPortal(
      <>
        {this.renderPreviewBanner()}
        {this.renderFirstRunOverlay()}
        {this.renderOnboarding()}
        {this.renderSnippets()}
      </>,
      this.footerPortal
    );
}
```

Reply:
much better 👍